stable/app/templates/cron.yaml

```diff
@@ -35,7 +35,15 @@ spec:
               image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag | default $.Chart.AppVersion }}"
               imagePullPolicy: {{ $.Values.image.pullPolicy }}
               command:
-              {{- toYaml $job.command | nindent 16 }}
+              {{- if $.Values.otelcollector.enabled }}
```

Comment:
sorry, we need to bump the version as well
https://github.com/goto/charts/blob/main/stable/app/Chart.yaml